.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file should not be in git

Reply:
IMHO it should. this is a "workspace setting" that means anyone working on this repo in VS Code will have format on save so that we don't have formatting issues in PRs etc.

Reply:
I see your point and I'm afraid I was unnecessarily terse in my comment above :) It's a good idea to enforce the formatting. Could we use https://github.com/prettier/eslint-plugin-prettier to make CI fail if your code is not formatted properly? This way we could leave all editor settings out of the repo and have people arrive at a clean build however they want.